portfolio-jsm/src/app/contact/page.tsx

```diff
@@ -1,8 +1,17 @@
+import ContactForm from "@/src/components/Contact";
+
 export default function Contact() {
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24 dark:text-white-800">
-      Contact
-      <div className="my-3">example</div>
+    <main className="flex min-h-screen flex-col items-center p-24 dark:text-white-800">
```

Comment:
The padding here is what is preventing you from extending the contact form out to the screen. Remember padding removes space on the INSIDE of the element.